components/react/modal.tsx

```
@@ -125,11 +131,11 @@
         );
       case ModalView.Connecting:
         let subtitle: string;
-        if (currentWalletData!.mode === 'wallet-connect') {
+        if (currentWalletData!?.mode === 'wallet-connect') {
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)